cmd/upgrade/executor/cstor_spc.go

```diff
@@ -50,11 +50,14 @@ func NewUpgradeCStorSPCJob() *cobra.Command {
 		Long:    cstorSPCUpgradeCmdHelpText,
 		Example: `upgrade cstor-spc --spc-name <spc-name>`,
 		Run: func(cmd *cobra.Command, args []string) {
+			util.CheckErr(options.RunCStorSPCUpgradeChecks(cmd, args), util.Fatal)
 			options.resourceKind = "storagePoolClaim"
-			util.CheckErr(options.RunPreFlightChecks(cmd), util.Fatal)
```

Comment:
I don't need of passing 'cmd' to RunCStorSPCUpgradeChecks and RunCStorSPCUpgrade.. same may be the case with other checks.. can we remove that if not required

Reply:
Done